framework/widgets/ActiveField.php

```diff
@@ -800,4 +837,22 @@ protected function getInputId()
     {
         return $this->_inputId ?: Html::getInputId($this->model, $this->attribute);
     }
+
+    /**
+     * adds aria attributes to the input options
+     * @param $options array input options
+     */
+    protected function addAriaAttributes(&$options)
+    {
+        if ($this->addAriaAttributes) {
+
+            $options['aria-required'] =
+                (!isset($options['aria-required']) && $this->model->isAttributeRequired($this->attribute))
+                ? 'true' : 'false';
```

Comment:
I'm not sure about this particular case, according to specs
[aria-required](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute)
[aria-invalid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute)
need to be provided explicitly as `true` or `false`
when to use booleans instead of strings for example
```
$options = [
  'aria-required' => false,
];
```
can be confusing because `aria-required` won't be rendered at all which can be
treated as absence of `aria-required` attribute and not as the input value is actually not required.